### DISCUSSION TITLE
Select field tooltip translation

### DISCUSSION THREAD

**Discussion by nadeaumarc** [filamentphp/filament#10251] [Help] [answered]
I can't figure out how to translate the "Create option" and "Edit option" on the select form fields. I tried editing the filament-forms components.php file of the language I need but it didn't work. Any ideas?

![image](https://github.com/filamentphp/filament/assets/127355317/54e7038f-0361-42fe-a8bb-a38a4f75b6a9)

**Comment by leandrocfe** [comment, marked as answer]
I think you can use this
```php
->createOptionAction(fn (Action $action) => $action->label('New label'))
```

**Reply by nadeaumarc** [reply]
Yes it works, thank you appreciate it!
